src/cmd/linuxkit/run_qemu.go

```diff
@@ -547,7 +615,10 @@ func splitPublish(publish string) (publishedPorts, error) {
 }
 
 func buildQemuForwardings(publishFlags multipleFlag, containerized bool) (string, error) {
-	forwardings := "user"
```

Comment:
want to confirm my understanding: is this ok because of `qemuNetworkingDefault == user`?